index.html

```diff
@@ -22,26 +24,189 @@
+            </a>
           </li>
           <li>
-            <a href="https://github.com/OscarFM-dev"><img src="./images/github.png" alt="github logo"></a>
+            <a href="https://github.com/OscarFM-dev">
+              <img src="./images/header-section/github.png" alt="github logo">
+            </a>
           </li>
           <li>
-            <a href="https://twitter.com/OscarFMdev"><img src="./images/twitter.png" alt="twitter logo"></a>
+            <a href="https://twitter.com/OscarFMdev">
+              <img src="./images/header-section/twitter.png" alt="twitter logo">
+            </a>
           </li>
           <li>
-            <a href="https://www.instagram.com/oscarfmdev/"><img src="./images/peace.png" alt="peace logo"></a>
+            <a href="https://www.instagram.com/oscarfmdev/">
+              <img src="./images/header-section/peace.png" alt="peace logo">
+            </a>
           </li>
         </ul>
       </section>
+
+      <section class="works">
+
+        <div class="project">
+          <img src="./images/works-section/project1.png" alt="First project image">
+          <div class="text-content">
+            <h3 class="project-title">Tonic</h3>
+            <div class="project-data">
+              <p class="bold">CANOPY</p>
+              <p class="middot">&bull;</p>
+              <p>Back End Dev</p>
+              <p class="middot">&bull;</p>
+              <p>2015</p>
+            </div>
+            <p class="project-description">A daily selection of privately personalized reads;
+               no accounts or sign-ups required.</p>
+            <ul class="buttons-flex">
+              <li class="li-button">html</li>
+              <li class="li-button">css</li>
+              <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See Project</button>
+          </div>
+        </div> <!-- .project -->
+
+        <div class="project">
+          <img src="./images/works-section/project2.png" alt="Second project image">
+          <div class="text-content">
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <div class="project-data">
+              <p class="bold">CANOPY</p>
+              <p class="middot">&bull;</p>
+              <p>Back End Dev</p>
+              <p class="middot">&bull;</p>
+              <p>2015</p>
+            </div>
+            <p class="project-description">A daily selection of privately personalized reads;
+               no accounts or sign-ups required.</p>
+            <ul class="buttons-flex">
+              <li class="li-button">html</li>
+              <li class="li-button">css</li>
+              <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See Project</button>
+          </div>
+        </div> <!-- .project -->
+
+        <div class="project">
+          <img src="./images/works-section/project3.png" alt="Third project image">
```

Comment:
- [ ] Kindly, order the images of your design to look similar to the Figma template. :octocat: